addons/cookoff/stringtable.xml

```diff
@@ -188,5 +188,41 @@
             <Korean>유폭 중 모든 탄약을 제거합니다.</Korean>
             <Japanese>誘爆によって全ての弾薬を除去します。</Japanese>
         </Key>
+        <Key ID="STR_ACE_CookOff_disableProjectiles_tooltip">
+            <English>Disables projectiles firing from the vehicle</English>
+            <French>Désactive le tir de projectiles depuis le véhicule</French>
+            <Spanish>Desactiva el disparo de proyectiles desde el vehículo</Spanish>
+            <Italian>Disattiva il fuoco dei proiettili dal veicolo</Italian>
+            <Russian>Отключает стрельбу снарядами из машины</Russian>
+            <German>Deaktiviert das Abfeuern von Projektilen aus dem Fahrzeug</German>
+            <Japanese>車両からの発射物の発射を無効にします</Japanese>
+        </Key>
+        <Key ID="STR_ACE_CookOff_disableSound_tooltip">
+            <English>Disables the 'popcorn' sound of projectiles cooking off inside the vehicle</English>
```

Comment:
They are not popcorn sounds; they are the cookoff sounds of projectiles. I don't think Popcorn is the correct wording here.

Reply:
How about 'popping' sound / noise?

Reply:
> How about 'popping' sound / noise?

disregard